hcipy/optics/photonic_lantern.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+from ..optics import OpticalElement, Wavefront
+from ..field import Field, CartesianGrid, RegularCoords
+from ..mode_basis import ModeBasis
+
+class PhotonicLantern(OpticalElement):
+    '''A generic photonic lantern
+
+    Parameters
+    ----------
+    lantern_modes : ModeBasis
+        The modes corresponding to the lantern ports.
+    wavelength : scalar
+        The wavelength of the simulation.
+    '''
+    def __init__(self, lantern_modes, normalize_modes=True):
```

Comment:
`normalize_modes`? It's not used by the function, and it's not in the docstring.